incubator/blockchain-client-openliberty/templates/default/README.md

```diff
@@ -0,0 +1,96 @@
+    FABRIC_WALLET_PROFILE={"type":"IN_MEMORY"}
+    FABRIC_WALLET_CREDENTIALS=[{"cert":"LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNXVENDQWYrZ0F3SUJBZ0lVTVQwa2JuZlRnaFR1S1MvS2RySkI5YjU2ZlZFd0NnWUlLb1pJemowRUF3SXcKZnpFTE1Ba0dBMVVFQmhNQ1ZWTXhFekFSQmdOVkJBZ1RDa05oYkdsbWIzSnVhV0V4RmpBVUJnTlZCQWNURFZOaApiaUJHY21GdVkybHpZMjh4SHpBZEJnTlZCQW9URmtsdWRHVnlibVYwSUZkcFpHZGxkSE1zSUVsdVl5NHhEREFLCkJnTlZCQXNUQTFkWFZ6RVVNQklHQTFVRUF4TUxaWGhoYlhCc1pTNWpiMjB3SGhjTk1qQXdOREUxTVRjME9UQXcKV2hjTk1qRXdOREUxTVRjMU5EQXdXakJkTVFzd0NRWURWUVFHRXdKVlV6RVhNQlVHQTFVRUNCTU9UbTl5ZEdnZwpRMkZ5YjJ4cGJtRXhGREFTQmdOVkJBb1RDMGg1Y0dWeWJHVmtaMlZ5TVE4d0RRWURWUVFMRXdaamJHbGxiblF4CkRqQU1CZ05WQkFNVEJXRmtiV2x1TUZrd0V3WUhLb1pJemowQ0FRWUlLb1pJemowREFRY0RRZ0FFZEZ0ZEZTVk4KcXd3TGU1ajZRMVVqd056eXhGbXpuK3pTbzRGbEZFdVplR0FJZER0OGQwcENjLytxRFNCZlB4SVREeEpONWt5ZAo2bHdhTmoxYnFObHhDYU43TUhrd0RnWURWUjBQQVFIL0JBUURBZ2VBTUF3R0ExVWRFd0VCL3dRQ01BQXdIUVlEClZSME9CQllFRkdGTEZPWUJJdWNBRGFTRHk3emdadm5IZWJKak1COEdBMVVkSXdRWU1CYUFGQmRuUWoycW5vSS8KeE1VZG4xdkRtZEcxbkVnUU1Ca0dBMVVkRVFRU01CQ0NEbVJ2WTJ0bGNpMWtaWE5yZEc5d01Bb0dDQ3FHU000OQpCQU1DQTBnQU1FVUNJUURkUFpsajJ3KzBZdXArV1JxSEtNaWI4QmdJdHg5VTBLVk9rUGlveHVtSnhnSWdmQi9iCnVpbFl4d3RXMUN0bENLYXY1Z29pWnRhbHEzbi9hbHpVbVpSMjRqTT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQ==","msp_id":"Org1MSP","name":"admin","private_key":"LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JR0hBZ0VBTUJNR0J5cUdTTTQ5QWdFR0NDcUdTTTQ5QXdFSEJHMHdhd0lCQVFRZ3FCNHBRNE5PRFZzT2RMR2MKYWxSM3NCcm9WM095aSs2MGpDdVpZK001blF1aFJBTkNBQVIwVzEwVkpVMnJEQXQ3bVBwRFZTUEEzUExFV2JPZgo3TktqZ1dVVVM1bDRZQWgwTzN4M1NrSnovNm9OSUY4L0VoTVBFazNtVEozcVhCbzJQVnVvMlhFSgotLS0tLUVORCBQUklWQVRFIEtFWS0tLS0t"}]
+    FABRIC_CHANNEL=mychannel
+    FABRIC_CONTRACT=myasset
+    FABRIC_DEFAULT_IDENTITY=admin
+
+## WalletManagement
+
+Wallet(s) are a concept in the fabric SDK that contain identities for transacting on your fabric network. To remain consistent with the terminology we have created the a WalletManager class that contains code for easily onboarding fabric identities into the service.  The WalletManager is intensionally extendable so other wallet implementations can easily be made use of. The purpose of this component is house all the fabric identifies your service will want to transact with. The approach is entirely configuration driven, there are 2 modes of operation out of the box.
+
+`FABRIC_WALLET_PROFILE={"type":"IN_MEMORY"}`
+
+`FABRIC_WALLET_PROFILE={"type":"FILE_SYSTEM","options":{"path":"/project/user-app/src/main/java/application/Org1"}}`
+
+Creating your own Wallet backend is as simple as creating Wallet Implementation, adding configuration and adding a configuration type so the wallet manager can load the new implementation when configured.
+
+## API and Data Model
```

Comment:
This should be the first section, before wallets.

Reply:
Sounds good!